participants/1313e/simple_functions.py

```diff
@@ -10,3 +10,46 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
```

Comment:
The name of this is "simple_function", can you please keep it in a few lines of codes (<10)?